### PR TITLE
Fix the key for Span of Tracing in pipelinerun reconciler

### LINKED ISSUE #6783: Pipelinerun instead of Taskrun as tag key in Tracing span
# Expected Behavior
The tag key should have the correct value of pipelinrun
# Actual Behavior
image

# Steps to Reproduce the Problem

1. Start tracing, first tag is incorrect in pipelinerun reconciler.
2.
3.

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

```
(paste your output here)
```

- Tekton Pipeline version:

main branch
v0.48.0
```
(paste your output here)
```

### PR DESCRIPTION
**Pull request by khrm**
Fixes #6783

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: pkg/reconciler/pipelinerun/tracing.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func initTracing(ctx context.Context, tracerProvider trace.TracerProvider, pr *v
 	// Create a new root span since there was no parent spanContext provided through annotations
 	ctxWithTrace, span := tracerProvider.Tracer(TracerName).Start(ctx, "PipelineRun:Reconciler")
 	defer span.End()
-	span.SetAttributes(attribute.String("taskrun", pr.Name), attribute.String("namespace", pr.Namespace))
+	span.SetAttributes(attribute.String("pipelinerun", pr.Name), attribute.String("namespace", pr.Namespace))
```
Comment (khrm) [inline pkg/reconciler/pipelinerun/tracing.go:68]: Should I change this to a const value? We might need to change this in Taskrun reconciler also if I do that.

Comment (Yongxuanzhang) [reply pkg/reconciler/pipelinerun/tracing.go:68]: I think this is ok. I'm not opposed to add a const though.

**Review by Yongxuanzhang — COMMENTED** [review]
/lgtm

File: pkg/reconciler/pipelinerun/tracing.go (hunk 1 above)
Comment (lbernick) [reply pkg/reconciler/pipelinerun/tracing.go:68]: you could use the constants from this file https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/controller.go

Comment (khrm) [reply pkg/reconciler/pipelinerun/tracing.go:68]: @lbernick These start with caps. The convention is to have it all in lowercase.

**Review by lbernick — APPROVED** [review]

**Pull request closed without merging**